package.json

```diff
@@ -1,52 +1,48 @@
 {
   "name": "lytics-ember-components",
-  "version": "0.4.2",
+  "version": "0.0.0",
+  "description": "The default blueprint for ember-cli addons.",
   "directories": {
+    "doc": "doc",
     "test": "tests"
   },
   "scripts": {
-    "start": "ember server",
     "build": "ember build",
-    "test": "ember test"
-  },
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/lytics/ember-components.git"
+    "start": "ember server",
+    "test": "ember try:testall"
   },
+  "repository": "",
   "engines": {
     "node": ">= 0.10.0"
   },
-  "maintainers": [
-    {
-      "name": "Michael Lange",
-      "email": "dingoeatingfuzz@gmail.com"
-    },
-    {
-      "name": "Steven Lindberg",
-      "email": "steven@lytics.io"
-    },
-    {
-      "name": "Christopher Thorn",
-      "email": "chris.thorn@lytics.io"
-    }
-  ],
```

Comment:
`ember init` also clobbered the `maintainers` field.  This will also need to be reverted, and feel free to add yourself.